duckduckgo/iOS
# Remove duplicate string

Task/Issue URL: https://app.asana.com/0/0/1204613695667261/f

**Description**: There were two localised string that had the same key. It changes the string key and corrects a spelling mistake.



**Steps to test this PR**:
1. Go to settings and to DuckDuckGo Windows App check the copy is unchanged.
2. Go to WindowsBrowserWaitlist.swift and on line 52 change isWaitlistRemoved to true
Test that the DuckDuckGo App for windows page shows the copy and feature indicated in https://app.asana.com/0/0/1204562778402691/f



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
